x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.test.tsx

```diff
@@ -1104,274 +1108,310 @@ describe('rules_list component with items', () => {
     expect(loadRuleAggregationsWithKueryFilter).toHaveBeenCalled();
   }
 
-  it('renders table of rules', async () => {
-    // Use fake timers so we don't have to wait for the EuiToolTip timeout
-    jest.useFakeTimers('legacy');
-    await setup();
-    expect(wrapper.find('EuiBasicTable')).toHaveLength(1);
-    expect(wrapper.find('EuiTableRow')).toHaveLength(mockedRulesData.length);
+  describe('render table of rules', () => {
```

Comment:
instead of one test that took over 4 seconds we split into smaller tests that won't be on working near the timeout limit